packages/create-styles/src/components/ThemeProvider/ThemeProvider.Context.js

```diff
@@ -0,0 +1,67 @@
+import { is } from '@wp-g2/utils';
+import { createContext, useContext, useRef } from 'react';
+
+import {
+	COLOR_BLIND_MODE_ATTR_PROP,
+	DARK_MODE_ATTR_PROP,
+	HIGH_CONTRAST_MODE_MODE_ATTR_PROP,
```

Comment:
Whoops 🙂 If you rebase this branch TypeScript will catch this error!

```suggestion
	HIGH_CONTRAST_MODE_ATTR_PROP,
```

Reply:
Thank you !! 😭 🙏